libs/datamodel/connectors/mongodb-datamodel-connector/src/lib.rs

```diff
@@ -40,14 +40,17 @@ impl MongoDbDatamodelConnector {
         }
 
         let mut bail = || {
-            errors.push_error(datamodel_connector::DatamodelError::ConnectorError {
-                    message: ConnectorError::from_kind(ErrorKind::FieldValidationError {
-                        field: field.name().to_owned(),
-                        message: "MongoDB `@default(auto())` fields must have `ObjectId` native type and use the `@id` attribute.".to_owned(),
-                    })
-                    .to_string(),
-                    span: field.ast_field().span,
-                });
+            let err = ConnectorError::from_kind(ErrorKind::FieldValidationError {
```

Comment:
It's a lot of work to construct these errors, would there be a way to make it easier with less lines of code? Don't need to do that in this PR though.

Some whitespace here would do miracles between the statements...

Reply:
Yep, I want to flatten all that to a single call.